engine/install/debian.md

```diff
@@ -127,7 +127,7 @@ Raspbian.
    > public key file before updating the package index:
    >
    > ```console
-   > $ sudo chmod a+r /etc/apt/keyrings/docker.gpg
+   > $ sudo chmod -R a+rx /etc/apt/keyrings
```

Comment:
I think this may be a bit too eager;

```bash
mkdir keyrings
touch keyrings/one.sh
touch keyrings/docker.gpg

ls -al keyrings/
total 8
drwxr-xr-x 2 root root 4096 Apr  8 11:41 .
drwxr-xr-x 3 root root 4096 Apr  8 11:41 ..
-rw-r--r-- 1 root root    0 Apr  8 11:41 docker.gpg
-rw-r--r-- 1 root root    0 Apr  8 11:41 one.sh
```

Notice that all files in the directory now have their executable bit set (including files we don't own);

```bash
chmod -R a+rx keyrings

ls -l keyrings/
total 8
drwxr-xr-x 2 root root 4096 Apr  8 11:41 .
drwxr-xr-x 3 root root 4096 Apr  8 11:41 ..
-rwxr-xr-x 1 root root    0 Apr  8 11:41 docker.gpg
-rwxr-xr-x 1 root root    0 Apr  8 11:41 one.sh
```

I think we need two steps;

- one to set the executable bit on the directory to make it browsable (using _capital_ `X` to only touch the directory itself)
- one to set the permissinos on the `docker.gpg` (we should not touch other files that we don't own)

```bash
mkdir keyrings2
touch keyrings2/one.sh
touch keyrings2/docker.gpg

chmod -R a+rX keyrings2
chmod -R a+r  keyrings2/docker.gpg

ls -al keyrings2/
total 8
drwxr-xr-x 2 root root 4096 Apr  8 11:41 .
drwxr-xr-x 4 root root 4096 Apr  8 11:41 ..
-rw-r--r-- 1 root root    0 Apr  8 11:41 docker.gpg
-rw-r--r-- 1 root root    0 Apr  8 11:41 one.sh
```